tests/integration/test_core.py

```diff
@@ -558,28 +561,32 @@ async def test_prometheus_integration(
     assert: prometheus metrics endpoint for prometheus is active and prometheus has active scrape
         targets.
     """
-    assert ops_test.model
-    prometheus: Application = await ops_test.model.deploy(
-        "prometheus-k8s", channel="stable", trust=True
+    await prometheus.relate(
```

Comment:
Should this also be in the/ another fixture? Similar for the other relations, especially since it looks like the relations are being cleaned up

Reply:
Changing to fixture with `yield`s to allow cleanups in fixture and not impact the test. Thank you for the comment, TIL!